pages/en/learn/command-line/run-nodejs-scripts-from-the-command-line.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Run Node.js scripts from the command line
+title: Run javascript code with Node.js from the command line
```

Comment:
I'm not sure if this change is adding much, and now the title will be different from the filename and also the URL slug